src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -3136,10 +3098,7 @@ void SmartScript::ProcessEvent(SmartScriptHolder& e, Unit* unit, uint32 var0, ui
         {
             if (!GetBaseObject())
```

Comment:
this might not be needed anymore as there are no more checks to do with GetBaseObject()